modules/cluster-logging-release-notes-5.0.0.adoc

```diff
@@ -0,0 +1,230 @@
+[id="cluster-logging-release-notes-5-0-0"]
+= OpenShift Logging 5.0.0
+
+This release includes Red Hat Bug Advisory, link:https://errata.devel.redhat.com/docs/show/66974[RHBA-2020:66974-04 Errata Advisory for Openshift Logging 5.0.0].
+
+[id="openshift-logging-5-0-new-features-and-enhancements"]
+== New features and enhancements
+
+This release adds improvements related to the following concepts.
+
+[discrete]
+[id="ocp-4-7-cluster-logging-renamed-openshift-logging"]
+==== Cluster Logging becomes Red Hat OpenShift Logging
```

Comment:
Is there a reason why this uses heading level 4 and not 3 like the other features in this section?

Reply:
Nice catch. Fixed it.